clientv3/integration/txn_test.go

```diff
@@ -74,7 +74,7 @@ func TestTxnWriteFail(t *testing.T) {
 
 	dialTimeout := 5 * time.Second
 	select {
-	case <-time.After(2*dialTimeout + time.Second):
+	case <-time.After(3*dialTimeout + time.Second):
```

Comment:
No. We control it. The timeout should be passed into gRPC, which is 5 seconds by default.